.docker/app/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ruby:2.6
+RUN \
+      wget -qO- https://deb.nodesource.com/setup_9.x | bash - && \
+      wget -qO- https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+      echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+      apt-get update -qq && \
+      apt-get install -y --force-yes --no-install-recommends \
+      nodejs yarn build-essential libpq-dev \
+      vim tmux \
```

Comment:
Do we need this?

Reply:
vim and tmux?   Yes. We need that in development mode.

Reply:
i.e. for convenience ... this is just for development mode...

Reply:
I disagree pretty strongly.  Docker should only run a single process.  If you need to edit, you can mount the volume and edit locally.

Reply:
@jcoyne editing mounted files in the container is not the use case.  It's when you need to edit a file that is not part of the mount . .i.e. one of the gems or any other file in the container you need to access that is not accessible via the mount.

Reply:
@jcoyne following up on this.  I followed up to your earlier objection with a clarifying statement but I haven't heard back.  I don't know if it's because you still object to the change or because you have not gotten around to it.  I can remove the tmux and vim if it's still objectionable to have them there despite my clarifying statement on my reasoning (I can always add them myself if I ever need them).

Reply:
Sorry to have dropped this.  Yes, I don't want to have vim and tmux.

Reply:
My experience is similar to @jcoyne's: I haven't added dev tools like vim and tmux to `Dockerfile` in other projects. I might be in favor of moving forward without them, and if that proves annoying going forward, we can revisit this?

Reply:
Thanks @jcoyne, @mjgiarlo, it's been removed.